### ISSUE TITLE
Collaps answers in answer page

### ISSUE THREAD

**Issue by simongogolin** [wepublic/ionic-app#18]
Im Antworten Bereich sollten die Antworten nach der Bewertung eingeklappt werden.
Und dann per Tap auch wieder ausgeklappt werden.
Dieses Issue würde ich aber hinten anstellen – ist nicht ganz so wichtig.

**Comment by ThyMYthOS** [comment]
Soll man die Antworten dann auch wieder öffnen können?

**Comment by SayAntiAnti** [comment]
Ja, genau. Wir hatten dafür diese kleinen Pfeilen vorgesehen (siehe screenshot). In diese sollten sich die Bubbles dann einklappen.

Ich denke wir klammern diesen Teil erst mal aus. Ich denke es gibt dringendere Issues.
**Also: Kann einfach so bleiben, wie es in der aktuellen Version der App ist.**


![bildschirmfoto 2017-08-18 um 21 47 42](https://user-images.githubusercontent.com/30910528/29475191-f27dfb68-845e-11e7-8e3e-69b02747007b.png)
